tests/tt_metal/tt_metal/unit_tests_fast_dispatch/common/command_queue_fixture.hpp

```diff
@@ -31,7 +31,9 @@ class CommandQueueFixture : public ::testing::Test {
     }
 
     void TearDown() override {
-        tt::tt_metal::CloseDevice(this->device_);
+        if (!getenv("TT_METAL_SLOW_DISPATCH_MODE")){
```

Comment:
don't think this is needed; you already have a check in `SetUp()`

Reply:
bc test_basic_pipeline (in the gtest_smoke suite) uses that fixtures, there's a possibility of it being run in SD mode when it should only be run in FD, if that happens it'll seg fault bc the teardown() is trying to close a device that was never initialized

Reply:
GTEST_SKIP() can't be called from within SetUpTestSuite or an environment, so will keep this the same